internal/pkg/webhooks/recording/recording.go

```diff
@@ -97,15 +100,16 @@ func (p *podSeccompRecorder) Handle(
 	items := profileRecordings.Items
 
 	for i := range items {
-		if items[i].Spec.Kind != "SeccompProfile" {
+		item := items[i]
+		if item.Spec.Kind != "SeccompProfile" {
 			log.Info(fmt.Sprintf(
-				"recording kind %s not supported", items[i].Spec.Kind,
+				"recording kind %s not supported", item.Spec.Kind,
```

Comment:
We should probably issue an event here too.

Reply:
Yeah I can follow-up on that. My idea was to enhance the unit testing of the webhooks in general so we can also add more debugging information with in that rush.

Reply:
sounds good.